clio-util/src/main/scala/org/broadinstitute/clio/util/auth/AuthUtil.scala

```diff
@@ -53,10 +77,11 @@ object AuthUtil extends ModelAutoDerivation with LazyLogging {
     for {
       jsonPath <- Either.cond(
         Files.exists(serviceAccountPath),
-        serviceAccountPath,
-        new RuntimeException(
-          s"Could not find service account JSON at $serviceAccountPath"
-        )
+        serviceAccountPath, {
+          val message = s"Could not find service account JSON at $serviceAccountPath"
+          logger.error(message)
+          new RuntimeException(message)
```

Comment:
OK, but seems a shame we gotta both log and then throw.

Reply:
The Exception isn't actually thrown. Its just there for the returned Either type.